client/i18n/locales/english/intro.json

```
@@ -409,8 +409,8 @@
           "This project will help you understand the fundamental concepts behind these algorithms, and how you can apply them to sort numerical data in web applications."
         ]
       },
-      "learn-intermediate-algorithmic-thinking-by-building-a-dice-game": {
-        "title": "Learn Intermediate Algorithmic Thinking by Building a Dice Game",
+      "review-algorithmic-thinking-by-building-a-dice-game": {
+        "title": "Review Intermediate Algorithmic Thinking by Building a Dice Game",
```

```suggestion
        "title": "Review Algorithmic Thinking by Building a Dice Game",
```

it seems you have removed the word Intermediate from everywhere else